docs/rfcs/2023-10-12-desktop-calls.md

```diff
@@ -0,0 +1,34 @@
+# Desktop calls
+
+To make audio and video calls on desktop there are some options:
+- adapt [libwebrtc](webrtc.googlesource.com/) from Google which would be the most reliable, performant and seamless solution;
+- include some kind of WebView to the app via libraries;
+- implement a signaling server in the app and let users to use HTML page with WebRTC code that already exist for Android.
+
+## WebRTC lib
+
+To adapt libwebrtc we need to make SDK that is compatible with Java (JNI layer + desktop implementation of VideoCodecs and other features). There are two SDKs exist already: for Android and for Objective-C. Making Android SDK compatible with Java-only SDK gives a lot of problems and requires to have 5+ professional C++ developers with weeks/months for developing. Which is not something good.
+
+Another considered option was adopting Jitsi Java SDK, but it's state is not very clear, and in any case it is much more effort.
+
+## WebView
+
+Including WebView is possible but requires a lot of megabytes of storage to waste on such libs. Because only Chromium-like WebViews support WebRTC features. Which means 100+ MB to the package on top of 200+ MB now.
+
+## Standalone browser + WebRTC HTML page
+
+The last solution is what can give the most useful result: the same package size as before + already existent code which can be reused with small modifications + quality of result will depend on Chromium/Firefox/Safari devs (which is good, since they are interested in making all features working for everyone). 
+
+# Details of implementation
+
+Since the code for WebView has already written (https://github.com/simplex-chat/simplex-chat/tree/0e4376bada2d0c4ec2ade7f30b0048dc8b13abd8/apps/multiplatform/android/src/main/assets/www) it can be used to make calls on a desktop browser too. The only differences are these:
+- UI needs to be changed - buttons controlling calls should be added. For example, end call, disable camera/mic
```

Comment:
```suggestion
- UI needs to be changed - buttons controlling calls should be added. For example, end call, disable camera/mic. This will be added to separate HTML and JS files that would communication with the existing WebRTC JS code via the existing functional API.
```